content/automate/ManageIQ/System/Event/EmsEvent/Azure.class/virtualmachines_poweroff_endrequest.yaml

```diff
@@ -0,0 +1,14 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: virtualMachines_powerOff_EndRequest
+    inherits: 
+    description: 
+  fields:
+  - rel4:
+      value: "/System/event_handlers/event_action_refresh?target=ems"
+  - rel5:
+      value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_suspend&param="
```

Comment:
Hi @lfu @gmcculloug 


Azure does not actually support the instance *suspend* operation, so when the user suspends an Azure instance in ManageIQ, we actually stop it on the provider:
https://github.com/ManageIQ/manageiq-providers-azure/blob/master/app/models/manageiq/providers/azure/cloud_manager/vm/operations/power.rb#L12

This is why you receive a  ```virtualMachines_powerOff``` event when you "suspend" an Azure instance.

I am thinking we should just disable the suspend button and force the user to power it off instead, this would mirror the functionality in Azure.